src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -1,35 +1,43 @@
 package cromwell.engine.backend
 
-import java.nio.file.{Files, Paths}
-
-import com.typesafe.config.Config
-import cromwell.binding
-import cromwell.binding.WdlExpression.ScopedLookupFunction
+import com.typesafe.config.{ConfigException, Config}
 import cromwell.binding._
-import cromwell.binding.types.{WdlFileType, WdlMapType}
-import cromwell.binding.values.{WdlFile, WdlMap, WdlString, WdlValue}
 import cromwell.engine.ExecutionIndex.ExecutionIndex
 import cromwell.engine._
 import cromwell.engine.backend.Backend.RestartableWorkflow
 import cromwell.engine.backend.jes.JesBackend
-import cromwell.engine.backend.local.{LocalBackendCall, LocalBackend, LocalEngineFunctions}
+import cromwell.engine.backend.local.LocalBackend
 import cromwell.engine.backend.sge.SgeBackend
 import cromwell.engine.db.DataAccess
 import cromwell.engine.workflow.CallKey
 import cromwell.parser.BackendType
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Failure, Success, Try}
+import scala.util.Try
 
 object Backend {
   class StdoutStderrException(message: String) extends RuntimeException(message)
   def from(backendConf: Config): Backend = from(backendConf.getString("backend"))
-  def from(name: String) = name.toLowerCase match {
-    case "local" => new LocalBackend
-    case "jes" => new JesBackend
-    case "sge" => new SgeBackend
-    case doh => throw new IllegalArgumentException(s"$doh is not a recognized backend")
+  def from(name: String) = try {
+    name.toLowerCase match {
+      case "local" => new LocalBackend
+      case "jes" =>
+        new JesBackend {
+          conf //forces configuration resolution to fail now if something is missing
```

Comment:
I'm not sure I'm buying what you're selling here :) Why are you doing things this way?